$OpenBSD: patch-taglib_CMakeLists_txt,v 1.1 2014/02/07 12:29:24 dcoppa Exp $

Use the "LINK_PRIVATE" mode, instead of an empty LINK_INTERFACE_LIBRARIES
property, to get rid of an annoying warning related to Policy CMP0022

--- taglib/CMakeLists.txt.orig	Tue Oct  8 17:50:01 2013
+++ taglib/CMakeLists.txt	Fri Feb  7 12:15:00 2014
@@ -310,7 +310,7 @@ set(tag_LIB_SRCS
 add_library(tag ${tag_LIB_SRCS} ${tag_HDRS})
 
 if(ZLIB_FOUND)
-	target_link_libraries(tag ${ZLIB_LIBRARIES})
+	target_link_libraries(tag LINK_PRIVATE ${ZLIB_LIBRARIES})
 endif()
 
 set_target_properties(tag PROPERTIES
@@ -318,7 +318,6 @@ set_target_properties(tag PROPERTIES
   SOVERSION ${TAGLIB_SOVERSION_MAJOR}
   INSTALL_NAME_DIR ${LIB_INSTALL_DIR}
   DEFINE_SYMBOL MAKE_TAGLIB_LIB
-  LINK_INTERFACE_LIBRARIES ""
   PUBLIC_HEADER "${tag_HDRS}"
 )
 if(BUILD_FRAMEWORK)
